### ISSUE TITLE
Back end functionality and having forms work properly 

### ISSUE THREAD

**Issue by morganhowell95** [Guru-Developers/Guru-Online#10]

**Comment by morganhowell95** [comment]
@aila293 do we have this done yet? Do all the forms submit properly? Is the backend good to go also?

**Comment by ghost** [comment]
Yup. Forms all work, files attached to the forms work, all the backend of the functionality that's been specified so far is there. The only things I'm aware of that aren't done is file storage in production and sending notification emails in production.

**Comment by morganhowell95** [comment]
What's going on with file storage in production?

On Thursday, February 4, 2016, Kammy Liu notifications@github.com wrote:

> Yup. Forms all work, files attached to the forms work, all the backend of
> the functionality that's been specified so far is there. The only things
> I'm aware of that aren't done is file storage in production and sending
> notification emails in production.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Guru-Developers/Guru-Online/issues/10#issuecomment-179325275
> .

## 

Best regards,

Morgan J. Howell

---

+1 (704) 985-5705
mjhowell@live.unc.edu
http://morgan-howell.com

**Comment by ghost** [comment]
My impression is that I don't have the access to configure the Heroku server from here, which you need if I'm understanding these instructions correctly: 
https://www.railstutorial.org/book/user_microposts#sec-image_upload_in_production
 I mentioned this to Josh so maybe it's already been set up by now?
